test/app_web/live/tags_live_test.exs

```diff
@@ -0,0 +1,45 @@
+defmodule AppWeb.TagsLiveTest do
+  use AppWeb.ConnCase, async: true
+  alias App.{Item, Timer, Tag}
+  import Phoenix.LiveViewTest
+
+  @person_id 55
+
+  test "disconnected and connected render", %{conn: conn} do
+    {:ok, page_live, disconnected_html} = live(conn, "/tags")
+    assert disconnected_html =~ "Tags"
+    assert render(page_live) =~ "Tags"
+  end
+
+  # test "display tags on table", %{conn: conn} do
+  #   tag1 = add_tag(%{person_id: @person_id, text: "Tag1", color: "#000000"})
+  #   add_tag(%{person_id: @person_id, text: "Tag2", color: "#000000"})
+  #   add_tag(%{person_id: @person_id, text: "Tag3", color: "#000000"})
+
+  #   {:ok, page_live, _html} = live(conn, "/tags")
+
+  #   assert render(page_live) =~ "Tags"
+
+  #   assert page_live
+  #          |> element("td[data-test-id=text_#{tag1.id}")
+  #          |> render() =~
+  #            "Tag1"
+  # end
```

Comment:
@nelsonic can you help me with this test?

I don't know why, but the add_tag is not adding the tags before the asserts below.

So it seems the rendered pages use the default tags created for the test database (shopping, breakfast, etc)

Can you take a look?

Reply:
done!